src/components/Chart/DonutChartStory.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { patternfly } from '../../common/patternfly';
+import { DonutChart } from './index';
+
+/**
+ * DonutChart constants
+ */
+
+const donutConfigData = {
+  columns: [['MHz Used', 60], ['MHz Available', 40]],
```

Comment:
This seems to be rendering "60% Mhz" in the title... and I think it should be "60 Mhz".

Reply:
fixed :-)

p.s.
Patternfly is very opinionated about that :-)
https://github.com/patternfly/patternfly/blob/master/src/js/patternfly-settings-charts.js#L24